flagengine/features/models.go

```diff
@@ -74,6 +80,16 @@ func (fs *FeatureStateModel) Value(identityID string) interface{} {
 	return fs.RawValue
 }
 
+func (fs *FeatureStateModel) IsHigherSegmentPriority(other *FeatureStateModel) bool {
+	if fs.FeatureSegment == nil {
+		return false
+
```

Comment:
Seems like odd formatting here (additional blank line)? Do we have any formatting checks as part of the workflow?

Reply:
Nope, we don't, although it's valid according go fmt